### PR TITLE
Feature/emoji picker

### PR DESCRIPTION
**Pull request by fepe999**
#### :link: Trello board reference:

- [Envío de emojis](https://trello.com/c/vvE5QmVD)

---

#### :pencil2: Description:
This PR adds the ability to select emojis and send them!

---

#### :movie_camera: Screen record:

<p align="center">

</p>

---

#### :warning: Warnings:

-Some emojis are not displayed correctly due to incompatibility issues with google chrome

---

### CODE REVIEW

File: src/components/ChatInput/index.jsx [hunk 1]
```diff
@@ -44,6 +46,11 @@ const ChatInput = () => {
     dispatchMessages(setNewMessage(e.target.value))
     handleTyping()
   }
+  const emojiParser = inputRef => code => {
+    const emoji = String.fromCodePoint(`0x${code}`)
```
Comment (eb16) [inline src/components/ChatInput/index.jsx]: Do we have a Hackerman 'round here ? 😎 😆

File: src/index.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const setAuthToken = token => {
   HttpService.defaults.headers.common['Access-Token'] = token
 }
 
-setAuthToken('meiyyyWftLLqmjchLhiKZzVJ')
+setAuthToken('M4GfaWmLQk1fDdMjHVitCjDk')
```
Comment (eb16) [inline src/index.jsx]: Why did you change the auth token?

Comment (fepe999) [reply src/index.jsx]: This is the one that we use on frontend dashboard
